ansible/Makefile

```diff
@@ -1,3 +1,7 @@
+ANSIBLE_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+CAILLEACH_DIR:=$(abspath ${ANSIBLE_DIR}/../../cailleach)
```

Comment:
Cailleach is private and most probably will continue to be private. So, this wont work for third parties. If we are okay with this for now, fine with me. But eventually, we should come up with a way to feed all the environment parameters into the logic in a more agnostic way.

Reply:
I can imagine having something else outside this repo describing the individual environment-specific parameters, but the environment-independent logic (and docs on the available parameters) should probably live here.

Reply:
I agree with @flokli and if we don't want to tell people that we call our repository `cailleach`, I guess we could rename the variable to something like `ENV_CONFIG_DIR`. With that I would image we will leave a comment clarifying that this means cailleach for wire employees, which defeats the purpose. So, I thought it was best to name the variable `CAILLEACH_DIR`.

About the docs, for now, I have put the docs in [terraform/README.md](https://github.com/wireapp/wire-server-deploy/tree/akshaymankar/sft-metrics-discovery/terraform/README.md).

Reply:
I think, we don't agree here. The *"more agnostic way"* that I had in mind would be sth like [this](https://github.com/wireapp/wire-server-deploy/pull/337/files#r487809617)

Note the `?` to allow overriding it. This way, the structure in cailleach can be whatever we want. The common denominator of us and third parties is the structure of an environment directory.

Reply:
hmm, making `ENV_DIR` overridable makes sense. I will do that. I will keep around the `CAILLEACH_DIR` variable, so people in our team don't have to set a separate `ENV_DIR` for each environment.

Reply:
Sorry, I meant: *I think, we don't disagree here.*

Anyway, sounds good. Thank you!